models/Document/DocType.php

```diff
@@ -52,7 +52,7 @@ class DocType extends Model\AbstractModel
      *
      * @var string
```

Comment:
```suggestion
```

Reply:
```suggestion
     * @var string|null
```
I'd rather go for string|null to keep it consistent with the rest.

Reply:
It shouldn't be redundant php docs. You can remove it also from the rest.